en/websites_privacy_notice.md

```diff
@@ -31,45 +32,49 @@ We may use cookies, clear GIFs, third party web analytics, device information, a
 {: #data-tools }
 
 * **Functionality**: We may use information such as cookies, device information, and IP addresses to enhance functionality of certain products, services, and communications. For example:
-    * We use cookies to remember information about your Firefox settings (such as language preference and Firefox add-ons), and so that we can recommend relevant features and Add-ons to you based on your Firefox settings. They also assist with user sign-in and authentication so you can bypass re-entering your passwords on certain Mozilla websites.
+    * We use cookies to remember information about your settings (such as language preference), and so that we can recommend relevant features and Add-ons to you based on your settings. They also assist with user sign-in and authentication so you can bypass re-entering your passwords on certain Mozilla websites.
```

Comment:
```suggestion
    * We use cookies to remember information about your settings (such as language preference), and so that we can recommend relevant features and add-ons to you based on your settings. They also assist with user sign-in and authentication so you can bypass re-entering your passwords on certain Mozilla websites.
```

No point in add-ons being uppercase.